src/pmpo_c.h

```diff
@@ -40,13 +40,14 @@ void polympo_setMeshGeomTypePlanar_f(MPMesh_ptr p_mpmesh);
 void polympo_setMeshGeomTypeSpherical_f(MPMesh_ptr p_mpmesh);
 void polympo_setMeshSphereRadius_f(MPMesh_ptr p_mpmesh, const double sphereRadius);
 void polympo_setMeshNumVtxs_f(MPMesh_ptr p_mpmesh, const int numVtxs);
-int polympo_getMeshNumVtxs_f(MPMesh_ptr p_mpmesh);
+void polympo_getMeshNumVertices(MPMesh_ptr p_mpmesh, int & numVtxs);
```

Comment:
See comment above about the suffix
```suggestion
void polympo_getMeshNumVertices_f(MPMesh_ptr p_mpmesh, int & numVtxs);
```

Reply:
Added all the changes. Please review before I resolve. Thanks.